onyo/shared_arguments.py

```diff
@@ -4,8 +4,22 @@
     action='append',
     type=str,
     help=r"""
-        Use the given **MESSAGE** as the commit message (rather than the default).
+        Append **MESSAGE** to the commit message.
         If multiple ``--message`` options are given, their values are
         concatenated as separate paragraphs.
     """
 )
+
+shared_arg_no_auto_message = dict(
+    args=('--no-auto-message',),
+    action='store_true',
+    help=r"""
+        Do not use auto-generated commit message subject.
+        This does not disable the inventory operations record
+        at the end of a commit message.
+        If there's also no **MESSAGE** given, the subject line
+        will be ``[Empty subject]``.
+        The default can be configured via 'onyo.commit.auto-message'
+        and is set to ``true`` by ``onyo init``.
```

Comment:
I propose:

> Do not auto-generate the commit message subject.
> If no **MESSAGE** is given, the subject line will be
> ``[Empty subject]``.
> The default can be configured via 'onyo.commit.auto-message'.

Reply:
I don't know if you intentionally resolved this without changes, or if it was an accident. Bumping this so you can say explicitly. :-)

Reply:
It is changed to what you said? I'm confused.

Reply:
Or do you want to remove the line about the operations record?

Reply:
Ahh, I was intentionally removing the information about `onyo init` (as suggested elsewhere) and the operations record.

Reply:
Ah, okay. I changed other parts of the wording, but kept this part, since I thought this useful when it's actually showing up in the command line help.

Reply:
(As opposed to python functions where it's indeed more a burden on maintenance than anything else)

Reply:
I'm fine with keeping the comment about the inventory operations. But the `init` comment should definitely be removed.

Otherwise, we should add that line to every other configuration option we have. It doesn't make sense. The config just "is" part of Onyo.